components/Clue/Clue.test.js

```diff
@@ -2,9 +2,9 @@ import { render, screen } from "@testing-library/react";
 import Clue from ".";
 
 test("displays two different symbols", async () => {
-  const randomColor = ["", "red", "blue"];
-  const randomSymbol = ["", "A", "B"];
-  render(<Clue randomColor={randomColor} randomSymbol={randomSymbol} />);
+  const randomColors = ["", "red", "blue"];
+  const randomSymbols = ["", "A", "B"];
+  render(<Clue randomColors={randomColors} randomSymbols={randomSymbols} />);
```

Comment:
Why is this test *async*? You're not using the `userEvent` methods here so it's not necessary, is it?

Reply:
If I remove async and await here, the test fails?🤨